resources/base.css

```diff
@@ -14,30 +14,64 @@ a:hover {
 }
 
 a.link-small {
-  font-size: 12px;
+  font-size: 13px;
 }
 
 ul {
   list-style: none;
+}
+
+ul.two-columns {
+  list-style: none;
   columns: 2;
 }
 
 ul > li {
   line-height: 26px;
+  padding: 0;
+}
+
+h1 {
+  font-size: 34px;
+}
+
+h2 {
+  font-size: 28px;
```

Comment:
Are these really necessary?  Are the defaults wrong in some way?

Reply:
We're not using default any way. I am making incremental changes on the visual structure of this demo depository in the future too. The style should be minimal, but also should be modern enough from my POV.